packages/hooks/src/useActiveSymbols.ts

```diff
@@ -0,0 +1,16 @@
+import { useFetch } from '@deriv/api';
+
+const useActiveSymbols = (mode?: 'brief' | 'full') => {
+    const res = useFetch('active_symbols', {
+        payload: {
+            active_symbols: mode || 'brief',
+        },
+        options: {
+            refetchOnWindowFocus: false,
+        },
+    });
+
+    return res;
+};
```

Comment:
```suggestion
/** A custom hook to get the list of active symbols. */
const useActiveSymbols = () => {
    const { data, ...rest } = useFetch('active_symbols', { payload: { active_symbols: 'brief' } });

    const active_symbols = useMemo(
        () =>
            data?.active_symbols?.map(symbol => ({
                ...symbol,
                /** The icon name for the underlying asset. */
                icon: `IcUnderlying${symbol.symbol}`,
            })),
        [data?.active_symbols]
    );

    return {
        /** List of active symbols. */
        data: active_symbols,
        ...rest,
    };
};
```
@amir-deriv Let's have the data transformation layer, Later trader team can make use of it based on their need since we are not aware of the requirements but we can give them the abstraction to start with.

Reply:
@maryia-deriv Just a heads-up of what's coming, Later trader team can start using this hook to get the active symbols only where it's needed inside the component lifecycle, By doing so you guys will be able to remove the related stores and logic inside of it. What @amir-deriv has done here is just pass the data from the hook to the existing stores since we can't make big breaking changes in one go and we need help from the trader team who knows the business.

That `icon` thing above is just an example of how we can transform BE response based on our need in a centralized top-level place, This helps to remove lots of small conditions in the codebase and make it easier to maintain changes. You can also have a look at the `useCurrencyConfig` hook for more examples, If still confusing let me know to have a talk 🙇🏻 

cc: @matin-deriv